tests/test_utils.py

```diff
@@ -3336,6 +3336,75 @@ def test_load_kasvs_ecdh_vectors():
     assert expected == load_kasvs_ecdh_vectors(vector_data)
 
 
+def test_load_kasvs_ecdh_kdf_vectors():
```

Comment:
Could you move this test + the small loader changes to a separate PR? (Sorry!) I'd like the added test vectors to be the only thing in this one so I can verify them and merge them independently of reviewing any code.